### DISCUSSION TITLE
Add manifest support to depend-info

### DISCUSSION THREAD

**Discussion by GordonSmith** [microsoft/vcpkg#27030] [Ideas]
****

**Is your feature request related to a problem? Please describe.**
`vcpkg.exe depend-info`  requires a package name and ignores local manifest

**Proposed solution**
Should behave in a similar way to `vcpkg.exe install` and build a graph of all the dependencies in the vcpkg.json manifest

**Describe alternatives you've considered**
Manually running `vcpkg.exe depend-info` for each item in my manfiest file and merging the responses

**Comment by moritz-h** [comment]
For me it worked by adding the project root directory with the vcpkg.json to the list of overlay port directories. Then I could just ask for the depend-info of the whole project at once.
```
cd project_root_dir
path/to/vcpkg depend-info --overlay-ports=project_overlay_ports --overlay-ports=. my_project
```
I'm running this command from the project root directory, because that way also the vcpkg-configuration.json with additional registries is also recognised automatically.

**Reply by AlexanderLanin** [reply]
This works perfectly, but it's not really intuitive ;-)
